themes/custom/ts_wrin/templates/blocks/block--menu-block--microsites.html.twig

```diff
@@ -6,28 +6,7 @@
  * Only show this block if it has a menu in it.
  */
 #}
-{% if content['#menu_name'] %}
```

Comment:
We still need this old code for specific pages. See
https://pr-1223-wriflagship.pantheonsite.io/paying-for-paris/how-much-will-it-cost

but if you add another template called `block--menu-block--microsites-level-2.html.twig` and add this old code there, the page should look right.

Reply:
Brilliant!